jx/ui/FillText.jxml

```diff
@@ -1,7 +1,7 @@
 <Label xmlns="jx/ui">
 	<script>
 		<![CDATA[
-			function onAttr(delta, attr) {
+			function onAttr(attr, delta) {
 				if ( delta.text || attr.text
 					&& (delta.height || attr.height) ) {
```

Comment:
note to self, based on the indentation, most likely forgot to group `(delta.text || attr.text)`.
Not related to this PR.